pkg/database/database_env.go

```diff
@@ -0,0 +1,42 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package database
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+const (
+	dbConnectionErrStr string = "Could not create database connection string: %s must be set"
+
+	DbKindEnvVar     string = "DATABASE_KIND"
+	DbNameEnvVar     string = "DATABASE_DB"
+	DbUserEnvVar     string = "DATABASE_USER"
+	DbPasswordEnvVar string = "DATABASE_PASSWORD" // nosec G101 not a password
+	DbHostEnvVar     string = "DATABASE_URL"
+	DbPortEnvVar     string = "DATABASE_PORT"
+)
+
+var DbEnvVars = [...]string{DbKindEnvVar, DbNameEnvVar, DbUserEnvVar, DbPasswordEnvVar, DbHostEnvVar, DbPortEnvVar}
+
+// Reads database connection info from the environment variables and returns a map of variable name to value.
+func NewDatabaseEnvironment() (map[string]string, error) {
```

Comment:
Do we need this method to be exposed? Is this meant to be used outside of the `database` package?